src/controllers/webviewController.ts

```diff
@@ -52,17 +54,20 @@ export class WebviewPanelController implements vscode.Disposable {
             }
         ));
         this._panel.onDidDispose(() => {
+            this.statusView.hideRowCount(this.uri);
```

Comment:
Shouldn't disposing the panel dispose the whole status bar?

Reply:
it's handled by the onCloseDocument event